#!/usr/bin/env python
PACKAGE = "yahboomcar_bringup"
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Kp", double_t, 0, "Kp in PID", 1.5, 0, 10)
gen.add("Ki", double_t, 0, "Ki in PID", 0.3, 0, 10)
gen.add("Kd", double_t, 0, "Kd in PID", 0.2, 0, 10)
gen.add("linear_max", double_t, 0, "speed in limit", 0.4, 0, 1.0)
gen.add("angular_max", double_t, 0, "speed in limit", 2.0, 0, 5.0)
gen.add("linear_min", double_t, 0, "speed in limit", 0.0, 0, 1.0)
gen.add("angular_min", double_t, 0, "speed in limit", 0.0, 0, 5.0)
exit(gen.generate(PACKAGE, "driver_node", "PIDparam"))
